packages/eslint-plugin/tests/rules/no-useless-template-literals.test.ts

```diff
@@ -207,7 +207,7 @@ ruleTester.run('no-useless-template-literals', rule, {
 
     {
       code: noFormat`\`\${    'a' + 'b'    }\`;`,
-      output: `'a' + 'b';`,
+      output: `('a' + 'b');`,
```

Comment:
How come this test case gains parens? Is that just the `getWrappingFixer` being overly cautions?

Reply:
I also have this question. Feels like a bug to me?

Reply:
At first, I thought it would be good for the expression to be wrapped, but as you said, I think it's excessive and looks like a bug.

However, in order to restore the test code, the getWrappingFixer function must be modified. Would it be a good idea to modify this function?